website/templates/project/settings.mako

```diff
@@ -71,28 +71,37 @@
-                        </h5>
-                        <p data-bind="if: !disabled">
-                            <button data-bind="css: {disabled: !dirty()},
-                                               click: cancelUpdateCategory"
-                                    class="btn btn-default">Cancel</button>
-                            <button data-bind="css: {disabled: !dirty()},
-                                               click: updateCategory"
-                                    class="btn btn-primary">Change</button>
-                        </p>
-                        <span data-bind="css: messageClass, html: message"></span>
-
-                        <span data-bind="if: disabled" class="help-block">
-                            A top-level project's category cannot be changed.
-                        </span>
-                    </div>
+
+                    <div id="projectSettings" class="panel-body">
+                        <div class="form-group">
+                            <label>Category:</label>
+                            <select data-bind="attr.disabled: disabled,
+                                                         options: categoryOptions,
+                                                         optionsValue: 'value',
+                                                         optionsText: 'label',
+                                                         value: selectedCategory"></select>
+                            <span data-bind="if: disabled" class="help-block">
+                              A top-level project's category cannot be changed
+                            </span>
+                        </div>
+                        <div class="form-group">
+                            <label for="title">Title:</label>
+                            <input class="form-control" type="text" maxlength="200" data-bind="value: title,
+                                                                                                      valueUpdate: 'afterkeydown'">
+                            <span class="text-danger" data-bind="validationMessage: title"></span>
+                        </div>
+                        <div class="form-group">
+                            <label for="description">Description:</label>
+                            <textarea data-bind="value: description,
+                                             valueUpdate: 'afterkeydown'",
+                            class="form-control resize-vertical"></textarea>
+                        </div>
+                           <button data-bind="click: cancelAll"
+                            class="btn btn-default">Cancel</button>
+                            <button data-bind="click: updateAll"
+                            class="btn btn-success">Save changes</button>
+                            <br>
+                            <span data-bind="css: messageClass, text: message"></span>
```

Comment:
Rather than using a `<br>` the message should go in its own div. See examples from elsewhere in the codebase:

```html
                    <!-- Flashed Messages -->
                    <div class="help-block">
                        <p data-bind="html: message, attr: {class: messageClass}"></p>
                    </div>

```